### PR TITLE
Ma incidentis samples

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/metricsadvisor/azure-ai-metricsadvisor/samples/sample_incidents.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_incidents.py
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/samples/sample_incidents.py:10]: should we add an async sample?

File: sdk/metricsadvisor/azure-ai-metricsadvisor/samples/sample_incidents.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    6) METRICS_ADVISOR_ALERT_CONFIGURATION_ID - the ID of an alert configuration
+    7) METRICS_ADVISOR_INCIDENT_ID - the ID of an incident
+    8) METRICS_ADVISOR_ALERT_ID - the ID of an alert
+"""
+
+import os
+
+
+def sample_list_incidents_for_detection_configuration():
+    # [START list_incidents_for_detection_configuration]
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/samples/sample_incidents.py:33]: can we add the sphinx snippets to the client as well?

**Review by kristapratico — APPROVED** [review]
Might be good to add sample to samples README as well

File: sdk/metricsadvisor/azure-ai-metricsadvisor/samples/sample_incidents.py (hunk 2 above)
Comment (xiangyan99) [reply sdk/metricsadvisor/azure-ai-metricsadvisor/samples/sample_incidents.py:33]: I think the question is do we want to merge them before the UX study? :)

**Review by kristapratico — APPROVED** [review]

**Pull request merged**